### DISCUSSION TITLE
Block read receipt even after replying to a message

### DISCUSSION THREAD

**Discussion by krashish8** [tomer8007/whatsapp-web-incognito#39] [Q&A]
Whenever I reply to a message, the read receipts are sent for the previous messages (which were previously blocked).
Is this what we expect from this extension?
If yes, can the read receipts be blocked, even after I reply to a message unless I manually mark the messages as read?

**Comment by tomer8007** [comment]
Yes, this is by design since issue #12 . Unfortunately right now there is no easy way to block the read receipts even after you replay to a message. 

There is some sense to it, though, which is that if we want to mimic nature (i.e what would happen without using this extension), then there couldn't be a situation in which there is a replay but some of the previous messages are not read.

**Reply by krashish8** [reply]
Okay, thanks! Makes sense.

However, sometimes, I notice that if I reply to a message and simultaneously my friend sends a message, then in my friend's WhatsApp, the message he sent "before" is marked unread, although it should be marked as read since my message is sent after his message. Consider this:

My friend's WhatsApp:

```
                                    << [1] Friend, MsgOld [Read]
                                    << [2] Friend, MsgNew [Unread]
>> [3] Me, MsgNew
```

Here, the message [2] shall be marked as read, as my message [3] is sent after this. However, it is marked unread sometimes.
Is this a synchronization issue, and thereby difficult to fix?

**Comment by krashish8** [comment]
**One last question:**
I have seen some WhatsApp Mod APKs (not official, nor safe I guess) have the feature that I can read a message, and the sender of the message will get only a "single tick". And I can alter it to "double tick" or "blue ticks" as per my wish for every person. Is it realistic to be implemented using this WhatsApp Web extension? Is a single tick possible, even if I received the message?

**Reply by krashish8** [reply]
@tomer8007 With whatsapp-web beta, I suppose this should be possible on Whatsapp Web? :thinking: 
Any ideas on how to receive all messages as a single tick, read them, and later mark them as a double tick or blue tick, as per one's need?

**Reply by tomer8007** [reply]
Yes, this is probably possible to do with the multi-device version, maybe a matter of blocking ack or similar. If you want this as a feature, you can open an issue for this, but this is going to be low priority for me

**Comment by tomer8007** [comment]
Regarding your first question, I think that it's possible that message [2] is actually received _after_ you send message [3] - only with a timestamp that makes it appear _before_ message [3]. In this case the situation you described could happen, but I think you should be able to at least send the read receipt for message [2] afterwards.

Regarding your second question, I haven't looked at it deeply, but I can guess that it's something that is more possible to do on your phone than on the web because it's very possible that the message is marked as "received" when it gets to the phone and only then gets forwarded to the web. Can you maybe refer to the modded APKs?

**Reply by krashish8** [reply]
Okay, thanks for the reply. I haven't seen or used the modded APKs for a while, and maybe they are not available in the Play Store. Anyway, I'll try to refer to those if I find them.

**Comment by krashish8** [comment]
Okay, thanks! I'll first try to do this myself in my leisure hours. Will
open an issue later on, if this seems difficult to me.

On Tue, 7 Dec 2021 at 22:04, Tomer ***@***.***> wrote:

> Yes, this is probably possible to do with the multi-device version, maybe
> a matter of blocking ack or similar. If you want this as a feature, you can
> open an issue for this, but this is going to be low priority for me
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tomer8007/whatsapp-web-incognito/discussions/39#discussioncomment-1767262>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJNXNGTXDBKYBDVOYX5Q6E3UPYZSRANCNFSM4YCKMFOQ>
> .
>


-- 
Regards,
Ashish Kumar.

**Comment by tomer8007** [comment]
@krashish8 You can now select to not auto-send the receipts in the GUI.
